simulation/simulation_interface/src/conversions.cpp

```diff
@@ -232,15 +232,15 @@ void toMsg(const openscenario_msgs::ActionStatus & proto, openscenario_msgs::msg
 void toProto(
   const openscenario_msgs::msg::LaneletPose & pose, openscenario_msgs::LaneletPose & proto)
 {
-  proto.set_lanelet_id(pose.lanelet_id);
+  proto.set_lanlet_id(pose.lanelet_id);
```

Comment:
Why renamed this?

Reply:
I've changed `lanlet` to `lanelet` in #354. Is the `lanlet` correct word? If so, sorry for that.

Reply:
No, the lanelet is correct and the lanlet is a typo.

Reply:
Thank you! fixed in here. https://github.com/tier4/scenario_simulator_v2/pull/357/commits/1f8ce0894f6bc77d9cb558a17f03e44105d6f4dd